profiling/src/internal/stack_trace.rs

```diff
@@ -16,6 +16,7 @@ impl Item for StackTrace {
 
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
 #[repr(transparent)]
+#[cfg_attr(test, derive(bolero_generator::TypeGenerator))]
```

Comment:
Wondering about this.  This is an opaque type that represents an interned index in the profile.  If we ever want to use it as such, we would need a constraint connecting the two.  But maybe for fuzzing for now this is ok?

Reply:
Fair